### ISSUE TITLE
community: categorise "good first issue" and "help wanted" issues

### ISSUE THREAD

**Issue by myitcv** [play-with-go/play-with-go#305]
Some people (e.g. @maudel in https://github.com/play-with-go/play-with-go/issues/163#issuecomment-728126863)  have expressed interest in help with the project which is great. 

We should therefore categorise issues as "good first issue" and "help wanted" to sign-post folks who want to help.

**Comment by marcosnils** [comment]
SGTM! I guess this issue resolution is to clarify this on the contribution guidelines?

**Comment by myitcv** [comment]
Indeed. I created this more as a reminder to actually reclassify issues.
